src/main/webapp/resources/js/pages/analysis/tabular-preview.js

```diff
@@ -1,8 +1,6 @@
-import $ from "jquery";
```

Comment:
I am sitting here looking at this code and wondering why on earth we are actually parsing this ourself.  Fundamentally it should be a simple task, but is potentially a nasty task.  I don't think we should change this now, but we already have `XLSX` in the project that has built in (and heavily tested) functions for handling this stuff (I think).  Maybe we want to think about updating this later.

Reply:
For sure. Right now it just reuses the previous code